Backend/Database/DbSeeder.cs

```diff
@@ -24,18 +21,34 @@ public static (List<Project>, List<User>, List<Tag>) GenerateProjects(int count)
 
             Random random = new();
 
-            int index1 = random.Next(tags.Length);
-            int index2 = random.Next(tags.Length);
+            int skillsAmount = random.Next(6);
+            int interestsAmount = random.Next(6);
+
+            List<Tag> fakedTags = [];
+
+            List<string> skills = ["Data", "Art", "C#", "Tim", "JS", "Java"];
+            for (int j = 0; j < skillsAmount; j++)
+            {
+                int index = random.Next(skills.Count);
+                var tagFaker = new Faker<Tag>()
+                    .RuleFor(t => t.TagValue, v => skills[index])
+                    .RuleFor(t => t.IsSkill, v => true);
 
-            var tagFaker = new Faker<Tag>()
-                .RuleFor(t => t.TagValue, v => tags[index1])
-                .RuleFor(t => t.IsSkill, v => true);
+                fakedTags.Add(tagFaker.Generate());
+                skills.RemoveAt(index);
+            }
 
-            var tagFake2r = new Faker<Tag>()
-                .RuleFor(t => t.TagValue, v => tags[index2])
-                .RuleFor(t => t.IsSkill, v => false);
+            List<string> interests = ["Data", "Art", "C#", "Tim", "JS", "Java"];
```

Comment:
how about we make these seeded lists contain different values from skills list?

Reply:
I don't see the need or effect it will have on our features